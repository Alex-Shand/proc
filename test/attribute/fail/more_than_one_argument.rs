#[proc::attribute]
fn test(_first: (), _second: ()) -> proc::TokenStream {
    todo!()
}

fn main() {}
