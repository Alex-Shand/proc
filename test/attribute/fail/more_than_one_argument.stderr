error: The attribute function should have one argument
 --> test/attribute/fail/more_than_one_argument.rs:2:1
  |
2 | fn test(_first: (), _second: ()) -> proc::TokenStream {
  | ^^
